apps/src/aichat/views/AichatView.tsx

```diff
@@ -32,30 +30,24 @@ import {
   onSaveComplete,
   onSaveFail,
   endSave,
+  addNotification,
 } from '../redux/aichatRedux';
-import {
-  AichatLevelProperties,
-  ChatCompletionMessage,
-  Role,
-  ViewMode,
-} from '../types';
+import {AichatLevelProperties, Notification, ViewMode} from '../types';
 import {isDisabled} from './modelCustomization/utils';
 import ChatWorkspace from './ChatWorkspace';
 import ModelCustomizationWorkspace from './ModelCustomizationWorkspace';
 import PresentationView from './presentation/PresentationView';
 import CopyButton from './CopyButton';
 import moduleStyles from './aichatView.module.scss';
 import aichatI18n from '../locale';
-import {getCurrentTime, getNewMessageId} from '../redux/utils';
+import {getNewMessageId} from '../redux/utils';
 
-const RESET_MODEL_NOTIFICATION: ChatCompletionMessage = {
+const getResetModelNotification = (): Notification => ({
```

Comment:
Good catch re: not having the right timestamp here! Looks like we're dumping the bold/non-bold here (making it all bold I think?), yeah? Seems ok to me, just confirming.

Reply:
Yeah good catch, it does show up as all bold. When we move the chat components to common components, we could look into possibly supporting markdown so we can preserve the original formatting?

Reply:
I wonder if this is more of a `ModelUpdate` type?